_includes/events-hero.html

```diff
@@ -0,0 +1,40 @@
+{% macro render(hero) %}
+
+{% from "post-macros.html" import category_icon as category_icon %}
+
+<section class="content_hero hero">
+    <div class="hero_wrapper">
+        <a href="#" class="hero_card hero_card__events">
```

Comment:
HTML  Doctor has a great writeup on `section` and apparently most of us have been using it wrong for some time http://html5doctor.com/the-section-element/